app/server/src/routes/projectRoutes.ts

```diff
@@ -15,6 +15,9 @@ export default {
         app.get('/project/:projectId',
             authCheck,
             controller.getProject);
+        app.patch('/project/:projectId/delete',
```

Comment:
It seems we are sending a request body with the patch request that deletes project samples. Delete does not normally have a body, so I guess that's why it uses patch.

Reply:
Yeah that's true, and also because it's not a straightforward delete of a resource. So that should stay as patch, but project delete should indeed be DELETE I think.